engine-precompiles/src/native.rs

```diff
@@ -242,9 +260,14 @@ impl Precompile for ExitToNear {
         // First byte of the input is a flag, selecting the behavior to be triggered:
         //      0x0 -> Eth transfer
         //      0x1 -> Erc20 transfer
-        let mut input = input;
         let flag = input[0];
-        input = &input[1..];
+        #[cfg(feature = "error_refund")]
+        let (refund_address, mut input) = parse_input(input);
+        #[cfg(not(feature = "error_refund"))]
+        let mut input = parse_input(input);
+        let current_account_id = String::from_utf8(sdk::current_account_id()).unwrap();
+        #[cfg(feature = "error_refund")]
```

Comment:
The `cfg!` macro will work much better here and generally should be done if its inside a method or function. Doesn't cause confusion for my IDE too.

Reply:
In CLion and VSCode you can tell which features should be considered active and which should not, so you can show relevant parts you are reviewing or working with.

Reply:
I know that, but thats not exactly the issue I'm having with CLion. The issue is that it thinks its always the former.

Reply:
I was under the impression that `cfg!` would be less efficient because it evaluates to a `bool` and both branches would exist in the code, whereas `#[cfg]` literally changes what code is shown to the compiler and therefore has zero runtime overhead.

I agree something using `cfg!` might be easier to read, but hopefully we won't have this `error_refund` feature around for very long. It can be made active by default once @mfornet and @sept-en do the necessary state migration, and then can be removed entirely after that.

Reply:
Fair. Then lets leave it in for now, resolve the other point outs.